build/docs.js

```diff
@@ -0,0 +1,25 @@
+/*
+	Build the docs
+*/
+
+var fs = require("fs");
+var path = require("path");
+var mkdirp = require('mkdirp');
+
+var file_names = ["./src/core.js", "./src/vector.js", "./src/special.js",
+                  "./src/distribution.js", "./src/linearalgebra.js", "./src/test.js"];
+var files = [];
+
+mkdirp('dist', function(err) {
+  if(err) {
+    console.error(err)
+  }
+});
+
+for(var i = 0; i < file_names.length; i++) {
```

Comment:
Simply because I mostly work on node, I'd say we start with that (uses eslint). Here's its config: https://github.com/nodejs/node/blob/master/.eslintrc

Doubt you were planning to, but don't worry about getting it setup for this PR.